src/server.py

```diff
@@ -97,7 +97,7 @@ def get_sound(self, uuid_):
         try:
             return self.sounds[uuid_]
         except KeyError:
-            self.logger.critical("This uuid is not assigned to any player.",
```

Comment:
I would squash "server: Rename "player" coincidences to "sound" in the log" into "server: Rename classes". It's part of the same rename.